atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -267,35 +227,17 @@ def run_in_orchestrator(self):
         else:
             if pullspecs:
                 replacement_pullspecs = self._get_replacement_pullspecs(pullspecs)
-                self._set_worker_arg(replacement_pullspecs)
                 related_images_metadata['pullspecs'] = replacement_pullspecs
             else:
                 # no pullspecs don't create relatedImages section
                 related_images_metadata['created_by_osbs'] = False
 
-        return operator_manifests_metadata
-
-    def run_in_worker(self):
-        """
-        Run plugin in worker. Replace image pullspecs based on replacements
-        computed in orchestrator, then create relatedImages sections in CSVs.
-
-        Exclude CSVs which already have a relatedImages section.
-        """
-        operator_manifest = self._get_operator_manifest()
-
-        if self._skip_all():
-            self.log.warning("skip_all defined, not running on worker")
-            return
-
         replacement_pullspecs = {
```

Comment:
you should use replacement_pullspecs which you've got previously

Reply:
that one is a list and the latest parts require a dictionary so this step converts it.